packages/vscode-extension/PRERELEASE.md

```diff
@@ -2,6 +2,63 @@
 
 ## Changelog
 
+### September 26, 2023
+
+#### New Features
+
+- **Sample App Gallery Search**: We have introduced the ability to search for sample apps within the Sample App Gallery.
+    ![Sample Gallery](https://github.com/OfficeDev/TeamsFx/assets/11220663/0b5636da-aa21-4624-82c1-2824f2557e9f)
+
+- **Custom Search Results Template Optimization**: We've optimized the `Custom Search Results` project template to seamlessly integrate with Microsoft 365 Copilot. To utilize this feature, please enable the `Develop Copilot Plugin` feature setting via Visual Studio Code in the [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings) and enroll in the Microsoft 365 Copilot [Early Access Program](https://aka.ms/PluginsEarlyAccess).
+    ![Copilot Plugin](https://github.com/OfficeDev/TeamsFx/assets/11220663/7fba4a92-f912-4021-b048-9f72de1267d8)
+
+#### New Additions to the Sample App Gallery
+
+- **SSO Enabled Tab via APIM Proxy**: This addition showcases how to build a single-page web application with a single sign-on feature leveraging Azure API Management service as a proxy to perform on-behalf-of flow, eliminating the need for a dedicated backend.
```

Comment:
We don't need to emphasize this is an addition. Can just do
addition-> sample app

same for the next sample.

Reply:
Updated